### ISSUE TITLE
Data processing and training details

### ISSUE THREAD

**Issue by hehao13** [snap-research/3dgp#4]
Thanks for your promising paper and code. I have the following questions regarding on the data processing and training scripts:

1. For the imagenet dataset, do we only need the training set or all the images (train, val, and test set)? 
2 What is the keep_ratio for imagenet dataset when selecting instances?
3 Could you provide the training scripts for other dataset, like SDIP Elephants?

**Comment by universome** [comment]
Hi @hehao13 !
1. For ImageNet, we only used the train set (we trained on the filtered version of its train set, and computed FID on the full train set).
2. For ImageNet, we kept 1/3 of the whole dataset, which constitutes 417139 images in total.
3. For other datasets, we used the same training script as for ImageNet, but with the default model size and with faster camera regularization annealing:
```
python src/infra/launch.py hydra.run.dir=. desc=MY_EXP_NAME dataset=elephant dataset.resolution=256 model.loss_kwargs.gamma=0.05 training.resume=null num_gpus=4 model.generator.camera_adaptor.emd.anneal_kimg=3000
```
